lib/getters/productGetters.ts

```diff
@@ -91,6 +91,7 @@ export const getProductFulfillmentOptions = (product: Product, purchaseLocation:
         : purchaseLocation?.name
         ? `${option.details}: ${purchaseLocation.name}`
         : "",
+    description: purchaseLocation?.name ? `Change Store` : `Select Store`,
```

Comment:
The Change Store and Select Store should be in a component somewhere. Getters should focus on getting existing properties from API / Data objects, not creating strings for UI use

Reply:
Done